Sources/App/routes.swift

```diff
@@ -1,98 +1,40 @@
-//            }
-    
+    }
     
-//------------------------------------------------------------------
-    
-//    router.get { req in
-//        return "Hello user, please consider going to /api/'EnterID'/'EnterUserName'/'EnterPassword'/'EnterProfileImageUrl'"
-//    }
-//        router.get("profile", String.parameter) { req -> String in
-//        let name = try req.parameters.next(String.self)
-//
-//        return "Welcome, \(name)!"
-//    }
-//    router.get("profileInfo") { req -> String in
-//            let xanderProfile = ProfileData(id: 1, username: "Xman1998", password: "notSoHidden", profileImage:"https://www.seekpng.com/png/detail/506-5061704_cool-profile-avatar-picture-cool-picture-for-profile.png")
-//        return "\(xanderProfile)"
-//    }
-//    router.post(ProfileData.self, at: "profileInfo") { req, data -> ProfileResponse in
-//
-//        return ProfileResponse(request: data)
-//    }
-//
-//    router.get("api", "id", String.parameter, "username", String.parameter, "password", String.parameter, "profileImage", String.parameter) { req -> String in
-//      let id = try req.parameters.next(String.self)
-//      let username = try req.parameters.next(String.self)
-//      let password = try req.parameters.next(String.self)
-//      let profileImage = try req.parameters.next(String.self)
-//      return "Your profile information is: \n ID:\(id)\n Username:\(username)\n Password:\(password)\n Profile Image:\(profileImage)"
-//    }
-//
-//    router.post(ProfileData.self, at: "api/name") { req, data -> String in
-//        return "Hello \(data.username) \(data.profileImage ?? "")"
-//    }
+    router.post("send") { req -> Future<User> in
+        let username: String = try req.content.syncGet(at: "username")
+        let password: String = try req.content.syncGet(at: "password")
+        let profileImage: String = try req.content.syncGet(at: "profileImage")
+        let fullName: String = try req.content.syncGet(at: "fullName")
+        let email: String = try req.content.syncGet(at: "email")
+        let gender: String = try req.content.syncGet(at: "gender")
+        let age: Int = try req.content.syncGet(at: "age")
+        let favouriteCount: Int = try req.content.syncGet(at: "favouriteCount")
+        let genrePref: String = try req.content.syncGet(at: "genrePref")
+        let userObj = User(id: nil, username: username, password: password, profileImage: profileImage, fullName: fullName, email: email, gender: gender, age: age, favouriteCount: favouriteCount, genrePref: genrePref)
+        return userObj.save(on: req)
+    }
     
+        router.post("authenticate") { req -> Future<[User]> in
+        let username: String = try req.content.syncGet(at: "username")
+        let password: String = try req.content.syncGet(at: "password")
+        let something = User.query(on: req).filter(\User.username == username).filter(\User.password == password).all()
```

Comment:
Name should be fixed. Try using foundUser or user instead of something

Reply:
I completely forgot about that var, thanks for reminding me. (was testing around at some point).